src/main/java/com/opencart/steps/EditAccountPageBL.java

```diff
@@ -0,0 +1,94 @@
+    }
+
+    public EditAccountPageBL editInformation() {
+        EditAccountModel editAccountModel = EditAccountModelRepository.getEditAccountModel();
+        inputFirstName(editAccountModel.getFirstName());
+        inputLastName(editAccountModel.getLastName());
+        inputEmail(editAccountModel.getEmail());
+        inputTelephone(editAccountModel.getTelephone());
+
+        clickOnContinueButton();
+
+        myAccountPage = new MyAccountPage();
+        return this;
+    }
+    public EditAccountPageBL cancelEditInformation() {
+        EditAccountModel editAccountModel = EditAccountModelRepository.getEditAccountModel();
+        inputFirstName(editAccountModel.getFirstName());
+        inputLastName(editAccountModel.getLastName());
+        inputEmail(editAccountModel.getEmail());
+        inputTelephone(editAccountModel.getTelephone());
+
+        clickOnBackButton();
+
+        myAccountPage = new MyAccountPage();
+        return this;
+    }
+    private void inputFirstName(String firstName) {
+        editAccountPage.getFirstNameInput().clear();
+        editAccountPage.getFirstNameInput().sendKeys(firstName);
+    }
+
+    private void inputLastName(String lastName) {
+        editAccountPage.getLastNameInput().clear();
+        editAccountPage.getLastNameInput().sendKeys(lastName);
+    }
+
+    private void inputEmail(String email) {
+        editAccountPage.getEmailInput().clear();
+        editAccountPage.getEmailInput().sendKeys(email);
+    }
+
+    private void inputTelephone(String telephone) {
+        editAccountPage.getTelephoneInput().clear();
+        editAccountPage.getTelephoneInput().sendKeys(telephone);
+    }
+
+    private void clickOnBackButton() {
+        editAccountPage.getBackButton().click();
+    }
+
+    public void clickOnContinueButton() {
+        editAccountPage.getContinueButton().click();
+    }
+
+    public void emptyFirstName(){
+        String expectedMessage = "First Name must be between 1 and 32 characters!";
+        Assert.assertEquals(editAccountPage.getErrorMessage().getText(), expectedMessage);
+    }
+    public void invalidLastName(){
+        String expectedMessage = "Last Name must be between 1 and 32 characters!";
+        Assert.assertEquals(editAccountPage.getWarningMessage().getText(), expectedMessage);
+    }
+    public void incorrectEmail(){
+        String expectedMessage = "E-Mail Address does not appear to be valid!";
+        Assert.assertEquals(editAccountPage.getCautionMessage().getText(), expectedMessage);
+    }
+    public void lackOfTelephoneCharacters(){
+        String expectedMessage = "Telephone must be between 3 and 32 characters!";
+        Assert.assertEquals(editAccountPage.getDangerMessage().getText(), expectedMessage);
+    }
+    public void redirectToPreviousPage() throws InterruptedException {
+        Assert.assertTrue(Boolean.parseBoolean(String.valueOf(myAccountPage.getAccountTab().isDisplayed())));
+    }
```

Comment:
All business logic  methods which are responsible for verifying something should start with "verify" or "check" prefix